src/components/sessions/SessionDetail.js

```diff
@@ -86,6 +86,9 @@ class SessionDetail extends React.Component {
               <Text style={{ color: style.colors.color1 }}>{fromTime} - {toTime}</Text>
             </View>
           </View>
+          <Button title="Feedback" 
+          {...this.props} 
+          onPress={() => this.props.navigation.navigate('Feedback', { sessionData: params.sessionData })}></Button>          
```

Comment:
We need to disable or hide this button until `devnull` accepts feedbacks. Atm it's configured to accept requests 10 minutes before the session ends. It might be a good idea to keep the client at for instance 8 or 5 minutes before the session ends to ensure that it's open.